gtk4/Gir.toml

```diff
@@ -1021,12 +959,6 @@ trust_return_value_nullability = false
     name = "add_choice"
     ignore = true  #strange type of options and option_labels
 
-[[object]]
-name = "Gtk.FileChooserButton"
```

Comment:
The file for this also needs to be removed or not? Also check if there are other files :)

Reply:
Yes, `FileChooserButton` no longer exists iirc

Reply:
Yes I mean the generated rs source file has to be removed from git, if that didn't happen already

Reply:
ah yes, i thought i did.